lib/TH/THFilePrivate.h

```diff
@@ -32,6 +40,7 @@ struct THFileVTable
     size_t (*writeLong)(THFile *self, long *data, size_t n);
     size_t (*writeFloat)(THFile *self, float *data, size_t n);
     size_t (*writeDouble)(THFile *self, double *data, size_t n);
+    size_t (*writeHalf)(THFile *self, half *data, size_t n);
```

Comment:
right, it was for the math. Yes it would be better to block Lua definitions also I just was not sure how to do it ...